src/modules/rules.ts

```diff
@@ -570,16 +567,14 @@ export const ruleCustomDataHandlers: {
 				if (MouseIn(1050, Y + 26 + i * 70, 766, 64)) {
 					DrawHoverElements.push(() => {
 						MainCanvas.save();
-						MainCanvas.fillStyle = "rgba(255, 255, 136, 0.9)";
-						MainCanvas.fillRect(1050, Y + 26, 766, 70 * PAGE_SIZE);
-						MainCanvas.strokeStyle = "Black";
-						MainCanvas.strokeRect(1050, Y + 26, 766, 70 * PAGE_SIZE);
+						DrawRect(1050, Y + 26, 766, 70 * PAGE_SIZE, "rgba(255, 255, 136, 0.9)");
+						DrawEmptyRect(1050, Y + 26, 766, 70 * PAGE_SIZE, "Black");
 						MainCanvas.textAlign = "left";
 						DrawTextWrap(msg + "   -   [click to copy into the empty input text field]", 1050 - 746 / 2, Y + 30, 756, 70 * PAGE_SIZE - 10, "black", undefined, 5);
 						MainCanvas.restore();
 					});
 				}
-				MainCanvas.strokeRect(1050, Y + 26 + i * 70, 766, 64);
+				DrawRect(1050, Y + 26 + i * 70, 766, 64, "black");
```

Comment:
This is a stroke, not fill:
```suggestion
				DrawEmptyRect(1050, Y + 26 + i * 70, 766, 64, "black");
```